events/README.md

```diff
@@ -0,0 +1,36 @@
+# Events
+
+```
+{
+  'name': string,
+  'description': string,
+  'room_id':string,
+  'template_id': string,
+  'enabled': boolean,
+  'lock': boolean,
+  'region': <'in' | 'us' | 'eu' | 'auto'>,
+  'peers': [
+    "<peer_id>",
+    "<peer_id>",
+    "<peer_id>"
+   ]
+  'created_by': {
+    ref: 'User',
+  },
+  'questions': [
+    "<question_id>",
+    "<question_id>",
+    "<question_id>"
+  ],
+  'comments: [
+    "<comment_id>",
```

Comment:
Do we have separate table for comments, peer and questions?

Reply:
Yes. Only the IDs are referenced here.